src/components/Board/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import App from './App'
+import List from './list'
```

Comment:
This should be '../List'

You also don't need to import `App`, `Card`, or `Button` (you never use them in this file)